airbyte-integrations/connectors/source-monday/acceptance-test-config.yml

```diff
@@ -18,10 +19,7 @@ acceptance_tests:
   discovery:
     tests:
       - config_path: "secrets/config.json"
-        # `boards`, `items`, `updates` streams schemas were modified
-        #  changed type of id for assets in items and updates
-        #  change deprecated owner to owners in boards
-        # All changes described in corresponding pr
+        # `boards`, `items`, `updates` streams schemas were modified. Pr: https://github.com/airbytehq/airbyte/pull/27410
```

Comment:
PR better write in upper case as acronym.

Reply:
Done